apps/store/themes/store/js/asset-core.js

```diff
@@ -4,8 +4,13 @@ var asset = {};
     var PROCESSING_TEXT = "processing";
     var SUCCESS_TEXT =  "success";
     var ERROR_TEXT = "error";
-    var getText = function(key) {
-        return $('#main-bookmark').data(key);
+    var getText = function (key) {
+        var bookmarkData = $('.main-bookmark');
+        var text = "";
```

Comment:
Add a message saying "A message was not defined for " + key + " please check process-asset templates"